proto/streamlit/proto/ForwardMsg.proto

```diff
@@ -134,3 +134,7 @@ message ElementDimensionSpec {
   // height in pixels
   uint32 height = 2;
 }
+
+message ForwardMsgList {
```

Comment:
Can you add a comment here about the purpose of the `ForwardMsgList`. Since it's unused within the Streamlit repo, it can be a bit confusing.